gusto/solvers/linear_solvers.py

```diff
@@ -383,6 +387,9 @@ def update_reference_profiles(self):
             logger.info('Compressible linear solver: Exner average solve')
             self.exner_avg_solver.solve()
 
+        self.hybridized_solver.invalidate_jacobian()
+        self.theta_solver.invalidate_jacobian()
```

Comment:
As only the right-hand side of the `theta_problem` depends on `thetabar` I think we can remove this

Reply:
removed